tests/ui/test_ui_tools.py

```diff
@@ -1106,6 +1106,7 @@ def pop_up_view(self, mocker):
         self.super_keypress = mocker.patch(VIEWS + '.urwid.ListBox.keypress')
         self.pop_up_view = PopUpView(self.controller, self.widgets,
                                      self.command)
+        return self.pop_up_view
```

Comment:
Are all these changes to support the `.sizing()` calls in the next commit? If so, we might want to structure it differently, or at least identify this in the commit text.

Reply:
Yes. I'll amend the commit text.